opm/autodiff/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -228,10 +252,24 @@ namespace Opm
             }
         }
 
-        void resize(const Wells* wells, size_t numCells, const PhaseUsage& pu)
+        void resize(const Wells* wells, size_t numCells)
```

Comment:
I am not familiar with the usage of this function, @andlaus , please comment if the change is not safe.

Reply:
I don't know anything here either. Maybe @atgeirr does?

Reply:
Okay. I thought this is a function you wrote. If not, never mind.